pkg/ossm/smcp_install.go

```diff
@@ -63,19 +63,19 @@ func TestSMCPInstall(t *testing.T) {
 		util.Shell(`oc get -n %s pods`, meshNamespace)
 	})
 
-	t.Run("smcp_test_uninstall_2.2", func(t *testing.T) {
+	t.Run("smcp_test_uninstall_2.3", func(t *testing.T) {
 		defer util.RecoverPanic(t)
-		util.Log.Info("Delete SMCP v2.2 in ", meshNamespace)
+		util.Log.Info("Delete SMCP v2.3 in ", meshNamespace)
 		util.KubeDeleteContents(meshNamespace, smmr)
 		util.KubeDeleteContents(meshNamespace, util.RunTemplate(smcpV22_template, smcp))
```

Comment:
This uninstall 2.3 step needs to run smcpV23_template.

Reply:
done!